lib/nightwatch-mocks-handler.js

```diff
@@ -44,14 +77,34 @@ _.extend(MocksHandler.prototype, {
       log('using mock server plugin');
       // adapter does most of the work
       var mockServer = this.mockServer = new Hapi.Server(this.mockServerOptions);
-      mockServer.connection({ port: this.mocksPort });
-      mockServer.register(this.mockServerPlugin, function (err) {
-        if (err) {
-          log('registered mock server plugin: ' + err.message);
-          return callback(err);
-        }
-        log('registered mock server plugin;  starting mock server');
-        mockServer.start(callback);
+
+      var httpConnectOptions = {
+        port: this.mocksPort,
+        labels: 'http'
+      };
+
+      var promises = [];
+      promises.push(startServer(mockServer, httpConnectOptions,this.mockServerPlugin));
+
+      if (this.mocksHttpsPort) {
```

Comment:
@jhudson8 
Will update it to check  keyFile and certFile. Thanks for pointing out.